src/components/EventModal/components/DropdownSelect.tsx

```diff
@@ -22,34 +24,38 @@ const DropdownIcon = ({
 
     
     return (
-        <Combobox
-            size="lg"
-            store={combobox}
-            onOptionSubmit={(val) => {
-                setDropDownValue(val)
-                onUpdate(val)
-                combobox.closeDropdown()
-            }}
-        >
-            <Combobox.Target>
-                <InputBase
-                    size="lg"
-                    component="button"
-                    type="button"
-                    pointer
-                    rightSection={<Combobox.Chevron />}
-                    rightSectionPointerEvents="none"
-                    onClick={() => combobox.toggleDropdown()}
-                >
-                    {dropDownValue || (
-                        <Input.Placeholder>{selectedOption}</Input.Placeholder>
-                    )}
-                </InputBase>
-            </Combobox.Target>
-            <Combobox.Dropdown>
-                {children}
-            </Combobox.Dropdown>
-        </Combobox>
+        <>
```

Comment:
was there a reason to wrap this in a empty fragment? otherwise it should just be reverted

Reply:
I experimented with adding a label but then saw that mantine supports it. I forgot to remove it. Now it should be gone.

Reply:
I experimented with adding a label but then saw that mantine supports it. I forgot to remove it. Now it should be gone.